docs/essentials/auto-generated-controls/ember.mdx

```diff
@@ -1,32 +1,30 @@
-To use auto-detected controls with Ember, you must fill in the `component` field in your story metadata:
+Storybook for Ember relies on [@storybook/ember-cli-storybook addon](https://github.com/storybookjs/ember-cli-storybook), to extract documentation comments from your component source files. If you're using Storybook with Ember, you should already have this addon installed, and you will just need to enable it by adding the following config block in your `ember-cli-build.js` file:
 
-```ts
-import { Button } from './Button';
-
-export default {
-  title: 'Button',
-  component: 'button', // name of your button component
-};
+```js
+let app = new EmberApp(defaults, {
+  '@storybook/ember-cli-storybook': {
+    enableAddonDocsIntegration: true,
+  },
+});
 ```
 
-Storybook uses this to auto-generate the `ArgTypes` for your component based on docgen information created by [ember-cli-addon-docs-yuidoc](https://github.com/ember-learn/ember-cli-addon-docs-yuidoc#documenting-components).
+Now, running the ember-cli server will generate a JSON documentation file at `/dist/storybook-docgen/index.json`. Since generation of this file is tied into the ember-cli build, it will get regenerated every time component files are saved. For details on documenting your components, check out the examples in the addon that powers the generation [ember-cli-addon-docs-yuidoc](https://github.com/ember-learn/ember-cli-addon-docs-yuidoc#documenting-components).
+
+Storybook uses this file to auto-generate the `ArgTypes` for your component based on docgen information created by [ember-cli-addon-docs-yuidoc](https://github.com/ember-learn/ember-cli-addon-docs-yuidoc#documenting-components).
```

Comment:
@crixx content wise it's almost good on my end. I've checked and one thing that stands. Let me elaborate on this. 
While checking this pull request i used an example that served as the code base for the Intro to Storybook tutorial for Ember, located [here](https://www.learnstorybook.com/intro-to-storybook/ember/en/get-started/), up to the Composite Component section to add some additional content to test this out. The thing here is that when running the script `storybook-dev` (`npm run storybook-dev`) which has the following inside:
```json
{
  "scripts":{
    "storybook-dev": "npm-run-all --continue-on-error --parallel start storybook",
   }
}
```
I get into this:
![issue-with-doc-gen-ember](https://user-images.githubusercontent.com/22988955/105400070-5a5fce80-5c1c-11eb-9da9-b10e042a1c86.png)

Basically it's constantly in a loop until i kill the terminal window.  One thing that struck me is that this only happens in MacOS, but on Windows it ran ok without any issues. The only way that i could bypass this issue in MacOs was first generate a build by running `npm run build` which is basically:

```json
"build": "ember build --environment=production",
```
And then issuing `npm run storybook` and no more errors in the console. 

I can "chalk this up" (pardon the bad pun) based on [this](https://cli.emberjs.com/release/basic-use/#additionalstepsformacandlinuxusers). But if we could adjust the docs and address this and avoid readers having to search for that i call it a win on our end?

What do you think?

Reply:
Jap, that is one of the classic Ember beginner's gotchas on Mac OSX. I completely agree that we should add it.